Gems/Terrain/Code/Source/EditorTerrainModule.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <EditorTerrainModule.h>
+
+namespace Terrain
+{
+    EditorTerrainModule::EditorTerrainModule()
+    {
+        m_descriptors.insert(
+            m_descriptors.end(),
+            {
+
```

Comment:
should the editor component be registered here or is that part of the next change?

Reply:
I removed it by accident as a part of splitting up my PRs.  It would end up in the next change anyways, but I'll add it in here.